fsspec/implementations/cached.py

```diff
@@ -684,7 +707,7 @@ def _check_file(self, path):
                 return fn
 
     def save_cache(self):
-        pass
+        self._cache_size = None
```

Comment:
Because of the size of the metadata file?

Reply:
Actually this isn't necessary, the tests pass without it.

Reply:
I have removed this now.